src/components/CookieAlert/CookieAlert.module.scss

```diff
@@ -0,0 +1,11 @@
+
+.cookieloc {
+    position: absolute;
```

Comment:
change this from `position: absolute;` to `position: fixed;`. On the register page, it doesn't stay on the bottom of the screen until you scroll down. This fixes that.